### DISCUSSION TITLE
Can not upload image files

### DISCUSSION THREAD

**Discussion by regul8or** [requarks/wiki#3417] [Help / Questions]
**Problem**
Can not upload image files.

**Steps**:

1. Open a page in the editor (markdown)
1. Click "Upload files" (it's a backward translation, may be something else)
1. System displays the red banner "An unexpected error occured"
1. Developer Tools/Network logs a POST /graphql request with 200 Ok status
1. In the "Upload files" section press "Browse", select a file. The selected file displayed in the uploadable files field
1. Press "Upload"
1. System displays a green banner "Upload complete" in the uploadable files field followed by "An unexpected error occured" red banner
1. Developer tools logs 2 more POST requests with 200 Ok status

**Host Info (please complete the following information):**
OS: Ubuntu (bionic) 18.04 x64
Wiki.js version: 2.4.107
Database engine: 5.7.30-0ubuntu0.16.04.1

![1](https://user-images.githubusercontent.com/21054394/88676522-d3122200-d0f4-11ea-8b61-6109e7e3f4fd.png)
![2](https://user-images.githubusercontent.com/21054394/88676524-d4434f00-d0f4-11ea-9611-dc6c33a8470b.png)
![3](https://user-images.githubusercontent.com/21054394/88676525-d4434f00-d0f4-11ea-868d-9e23d2b9b02b.png)

**Comment by danieltoro165051** [comment]
also happen me, any solution?

**Comment by etruriapa** [comment]
It happens to me too.
The error come from a POST to graphql.
Here is the error:
https://pastebin.com/zSKD7FzL

Using mysql 5.7.25-1 - (Debian)

**Comment by Smankusors** [comment]
oh I just realized, you guys are using MySQL 5.7. This feature is not supported if you are using MySQL 5.7. You need to upgrade to MySQL 8.x at least

Read more about it [here](https://docs.requarks.io/install/requirements/mysql5)

**Comment by danieltoro165051** [comment]
@Smankusors Not would applied for me. My db is postgress version
12.1

**Comment by etruriapa** [comment]
> oh I just realized, you guys are using MySQL 5.7. This feature is not supported if you are using MySQL 5.7. You need to upgrade to MySQL 8.x at least
> 
> Read more about it [here](https://docs.requarks.io/install/requirements/mysql5)

Thanks, I'll be moving the wiki to PostgreSql soon.
However, with MySQL 5.7, the upload does not even work in the root folder.

**Comment by wisetux** [comment]
> @Smankusors Not would applied for me. My db is postgress version
> 12.1

How are you accessing your Wiki? Are you running the instance behind some reverse proxy?

**Comment by regul8or** [comment]
> 
> 
> oh I just realized, you guys are using MySQL 5.7. This feature is not supported if you are using MySQL 5.7. You need to upgrade to MySQL 8.x at least
> 
> Read more about it [here](https://docs.requarks.io/install/requirements/mysql5)

Upgrading to MySQL 8 solved the issue for me. I'm feeling stupid

Now I wonder how to delete all that junk that was uploaded

**Comment by danieltoro165051** [comment]
> > @Smankusors Not would applied for me. My db is postgress version
> > 12.1
> 
> How are you accessing your Wiki? Are you running the instance behind some reverse proxy?

Hi, deployed with k8s and Ingress Controller.
